internal/services/eventhub/eventhub_resource.go

```diff
@@ -163,6 +164,29 @@ func resourceEventHub() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	if !features.FivePointOhBeta() {
+		r.Schema["namespace_id"] = &pluginsdk.Schema{
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ForceNew:     true,
```

Comment:
This probably shouldn't be `ForceNew` yet so that users can migrate over to using `namespace_id`